Ratings List done

adriantd3 opened the pull request:
Created the component that will be shown along with the comments section of an article.

It receives the article's avg rating and the list of ratings

adriantd3 committed:
Ratings List done

Created the component that will be shown along with the comments section of an article.

It receives the article's avg rating and the list of ratings

VctPerez commented:
![image](https://github.com/user-attachments/assets/bc7ac084-624c-482b-9fa2-f25400614819)
Yo añadiría esto al componente como hice con los comentarios y esta mas organizado.

adriantd3 committed:
Ratings section added

Made that the ratings list component shows the 100% value.

Also created a new component that shows the user input rating and the ratings list.

**IMPORTANT**
The ratings-section variables will need to be passed as props when the api integration works.
